services/QuillLMS/spec/workers/teacher_requested_verification_analytics_worker_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TeacherRequestedVerificationAnalyticsWorker do
+  subject { described_class.new }
+
+  describe '#perform' do
+    let!(:teacher) { create(:teacher) }
+    let(:analyzer) { double(:analyzer) }
+
+    before { allow(Analytics::Analyzer).to receive(:new) { analyzer } }
```

Comment:
Take or leave a purely stylistic thing, but I like the explicit `.and_return(analyzer)` over the block syntax `{ analyzer }`